script/playbackstrategy/msestrategy.js

```diff
@@ -332,7 +344,7 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
         mediaPlayer = dashjs.MediaPlayer().create();
         var playerSettings = Utils.merge({
           debug: {
-            logLevel: 2
+            logLevel: 4
```

Comment:
I assume you don't want this all the time.

Reply:
Good spot, thought I'd solved that with an earlier commit!